source/mainnet/net/resources/glossary.rst

```diff
@@ -487,6 +522,13 @@ way. The staked amount is proportional to the :ref:`lottery power<glossary-lotte
 
 :ref:`Delegators<glossary-delegate>` can delegate stake to a baker pool or passive delegation. This affects the staked amount of the baker and thus their lottery power.
 
+.. _glossary-statement:
+
+Statement
+=========
+
+One or more attributes to be proven or revealed by a wallet owner to a dApp or service.
```

Comment:
The part about "to be proven" does not read very well.

Maybe the following is better?
"A list whose items are either attributes to reveal, or properties of attributes to prove."